build/install-bundle.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+OPERATOR_SOURCE_INSTALL_FILES=(
+  "deploy/role.yaml"
+  "deploy/service_account.yaml"
+  "deploy/role_binding.yaml"
+  "deploy/crds/infinispan.org_infinispans_crd.yaml"
```

Comment:
we probably want also `clusterrole.yaml` and `clusterrole_binding.yaml` needed for the cross site feature?

Reply:
Yes, sure.
But got following error during install:
**The ClusterRoleBinding "infinispan-operator" is invalid: subjects[0].namespace: Required value**

This means that ClusterRoleBinding need to be defined namespace for account. This is limiting for the flexible install in any namespace/project

Reply:
It seems it's not possible to imply the current namespace in the subject. Really annoying!
I can see only 2 options:

- fix a namespace (at least for the serviceaccount)
- split the install in two steps

they are equally bad to me :-)

Reply:
Just another idea, tag this namespace with something specific, like $infinispan-operator-ns and replace with sed as I did in push-okd4.sh

Reply:
What do you think about:
- implement your approach in the `make install-bundle` to accept the custom namespace and sed all the .yaml for the users that need to specify namespace
- leave the yaml with fixed `infinispan-operator` project: this for users that want to quickstart with standard oc/kubectl commands

this to keep the example clean with only oc/kubectl commands for people that want to quickstart

Reply:
@rigazilla I'm prefer much simple use case as possible for this issue. Let's keep `make install-bundle` for joining files content only and add notification, that customer needs to replace this namespace with your own one. Strimzi project do this
https://strimzi.io/docs/operators/master/deploying.html#deploying-cluster-operator-str
wdyt?

Reply:
@dmvolod ok simple enough: let's leave `infinispan-operator` and add a note aside for users that want to change namespace.